packages/next/src/schematics/application/schema.json

```diff
@@ -61,6 +61,19 @@
         ]
       }
     },
+    "server": {
```

Comment:
What do you think about having a string property called server that points to the server file instead of an enum that differentiates between the default server and `server/server.dev.js`?

So your configuration would say:

```
"server": "server/server.dev.js"
```
So we use the provided server file or the default one if none is provided?

Reply:
Indeed will give more flexibility to the developer, but how he would suggest him to follow the function pattern:
```
/**
 * @param {DevServer} app
 * @param {{dev: string; dir: string; staticMarkup: boolean; quiet: boolean; conf: any; port: number;}} options
 * @returns {Promise<Server>}
 */
module.exports = async function nextServer(app, options) { ... }
```
Documenting? And template will be inline, writing to this suggestion file that dev gives?
What you think @vsavkin ? I would like to follow your guides and patterns.

Reply:
Hi @vsavkin made changes on configuration and learned how to create template files from template string. Take a look on it and let me know what is your opinion.